views/scss/qti/_choice.scss

```diff
@@ -40,6 +40,14 @@
         // ol
         .choice-area {
             margin-top: 25px;
+            .label-content {
+                display: inline-block;
+                .qti-block {
+                    img {
+                        width: 100%;
+                    }
+                }
+            }
```

Comment:
On top of that I found another potential issue: you can normally define the size in percentage of a picture in the QTI creator (in the property form on the right). But now even if you set a percentage, e.g. 25%, it always show 100%. Can you find a way to fix this ?